pages/platform/kubernetes-k8s/installing-artifactory/guide.en-gb.md

```diff
@@ -0,0 +1,224 @@
+
+In a terminal, configure the master key as follows:
+
+```bash
+$ export MASTER_KEY=$(openssl rand -hex 32)
+
+$ echo ${MASTER_KEY}
+```
+
+Output should be like this:
+
+<pre class="console">
+<code>
+$ export MASTER_KEY=$(openssl rand -hex 32)
+
+$ echo ${MASTER_KEY}
+xxxxxxxxfffffffffffffxxxxxxxxxxxxxxxxxxxxxgggggggggggggxxxxxxxxxx
+</code>
+</pre>
+
+Next, configure the join key as follows:
+
+```bash
+$ export JOIN_KEY=$(openssl rand -hex 32)
+
+$ echo ${JOIN_KEY}
+```
+
+Output should be like this:
+
+<pre class="console">
+<code>
+$ export JOIN_KEY=$(openssl rand -hex 32)
+
+$ echo ${JOIN_KEY}
+xxxxxxxxyyyyyyyyyyyyyxxxxxxxxxxxxxxxxxxxxxyyyyyyyyyyyyxxxxxxxxxx
+</code>
+</pre>
+
+### Install Jfrog Artifactory with Helm
+
+First, you need to create the `artifactory` namespace:
+
+```bash
+$ kubectl create ns artifactory
+```
+
+Next, you can install Jfrog Artifactory with the following helm command:
+
+```bash
+$ helm upgrade --install artifactory --set artifactory.masterKey=${MASTER_KEY} --set artifactory.joinKey=${JOIN_KEY} --namespace artifactory jfrog/artifactory
+```
+Output should be like this:
+
+<pre class="console">
+<code>
+$ helm upgrade --install artifactory --set artifactory.masterKey=${MASTER_KEY} --set artifactory.joinKey=${JOIN_KEY} --namespace artifactory jfrog/artifactory
+
+Release "artifactory" does not exist. Installing it now.
+NAME: artifactory
+LAST DEPLOYED: Wed Oct 12 10:41:49 2022
+NAMESPACE: artifactory
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+Congratulations. You have just deployed JFrog Artifactory!
+
+1. Get the Artifactory URL by running these commands:
+
+   NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+         You can watch the status of the service by running 'kubectl get svc --namespace artifactory -w artifactory-artifactory-nginx'
+   export SERVICE_IP=$(kubectl get svc --namespace artifactory artifactory-artifactory-nginx -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+   echo http://$SERVICE_IP/
+
+2. Open Artifactory in your browser
+   Default credential for Artifactory:
+   user: admin
+   password: password
+</code>
+</pre>
+
+Next, wait until the status of all Pods status are `Running` and _ready_ (i.e the number of Pods desired equals to the actual number in the `Ready` column):
+
+```bash
+$ kubectl get pods -n artifactory
+```
+
+Output should be like this:
+
+<pre class="console">
+<code>
+$ kubectl get pods -n artifactory
+NAME                                             READY   STATUS    RESTARTS   AGE
+artifactory-0                                    1/1     Running   0          8m11s
+artifactory-artifactory-nginx-7c556cb56b-x5wvm   1/1     Running   0          8m11s
+artifactory-postgresql-0                         1/1     Running   0          8m11s
+</code>
+</pre>
+
+### Test the fresh installed Jfrog artifactory
+
+Open the service URL in a browser and enter the default credentials `admin` and `password`:
```

Comment:
Ne pas oublier de mettre les commandes pour que l'utilisateur puisse le faire, sans avoir à réfléchir :)

```
export SERVICE_IP=$(kubectl get svc --namespace artifactory artifactory-artifactory-nginx -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
echo http://$SERVICE_IP/
```

Open the service URL in a browser and enter the default credentials: `admin` for *Username* and `password` for *Password*:

Reply:
done : https://github.com/ovh/docs/pull/3560/commits/8ff8516d12901f6dc6f717efb9ff61b0490b6266